### ISSUE TITLE
When using [filewords] to generate class images blank txt files are created

### ISSUE THREAD

**Issue by NagashSzarekh** [d8ahazard/sd_dreambooth_extension#415]
`
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: 44c46f0ed395967cd3830dd481a2db759fda5b3b
Installing requirements for Web UI

Checking Dreambooth requirements...
Checking torch and torchvision versions
Dreambooth revision is c2fc683ebcf4db23cfce22626aa6a7b565994c96
[+] bitsandbytes version 0.35.0 installed.
[ ] diffusers version 0.9.0 installed.
[+] transformers version 4.21.0 installed.
[+] torch version 1.12.1+cu116 installed.
[+] torchvision version 0.13.1+cu116 installed.
[!] xformers NOT installed.
`

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes

**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes

**Have you updated Dreambooth to the latest revision?**
Yes

**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Tes
**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this. Reply 'OK' Below to acknowledge that you did this.**
Ok

When using [filewords] to generate class images it is creating the images in the desired folder, but it is also leaving a empty txt file with the same name as the generated image. I assume the caption used to generate the image is supposed to be inserted into this file, but this does not appear to be happening. If this is not supposed be happening why leave behind an empty txt file.

From the image below you can see it is using the [filenames] to create the Class images
![image](https://user-images.githubusercontent.com/36505807/205528981-64559be3-c713-4944-bb50-e4826a17255f.png)

**Environment**

What OS?
Windows 10 Pro

If Windows - WSL or native?
Native
What GPU are you using?
Titan Xp
**Screenshots/Config**

[db_config.zip](https://github.com/d8ahazard/sd_dreambooth_extension/files/10149998/db_config.zip)

**Comment by ItsOkayItsOfficial** [comment]
We're getting the same behavior except that it's only writing the _first letter_ of the prompt in each file. Example attached.

```
Python 3.9.2 (tags/v3.9.2:1a79785, Feb 19 2021, 13:44:55) [MSC v.1928 64 bit (AMD64)]
Commit hash: 98947d173e3f1667eba29c904f681047dea9de90
Installing requirements for Web UI
Checking Dreambooth requirements...
Checking/upgrading existing torch/torchvision installation
Checking torch and torchvision versions
Dreambooth revision is c2fc683
[+] bitsandbytes version 0.35.0 installed.
[ ] diffusers version 0.9.0 installed.
[+] transformers version 4.21.0 installed.
[+] torch version 1.12.1+cu116 installed.
[+] torchvision version 0.13.1+cu116 installed.
[!] xformers NOT installed.
```
**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes

**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes

**Have you updated Dreambooth to the latest revision?**
Yes

**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes

**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this. Reply 'OK' Below to acknowledge that you did this.**
Ok

**What OS?**
Windows 11 Pro

**If Windows - WSL or native?**
Native

**What GPU are you using?**
RTX 3090 Ti

**Screenshots/Config**
[db_config.txt](https://github.com/d8ahazard/sd_dreambooth_extension/files/10156471/db_config.txt)

[0-5544fec9d783e5c4aac75c631685105961fc7888.txt](https://github.com/d8ahazard/sd_dreambooth_extension/files/10156374/0-5544fec9d783e5c4aac75c631685105961fc7888.txt)

**Comment by d8ahazard** [comment]
Fixed with [e7b2754](https://github.com/d8ahazard/sd_dreambooth_extension/commit/e7b2754a6c1917e1934256e9b4bb37464a80d67c)
